packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -380,8 +380,6 @@ abstract class ScrollableState<T extends Scrollable> extends State<T> {
   }
 
   void didUpdateScrollBehavior(double newScrollOffset) {
-    if (newScrollOffset == _scrollOffset)
```

Comment:
Maybe add the dartdocs for that function now, since you have the implications paged in. That way we can document the expected calling patterns, the costs of calling the function, etc.